common/lib/xmodule/xmodule/modulestore/store_utilities.py

```diff
@@ -3,6 +3,9 @@
 from collections import namedtuple
 
 import uuid
+from xblock.core import XBlock
+
+DETACHED_XBLOCK_TYPES = [name for name, __ in XBlock.load_tagged_classes("detached")]
```

Comment:
If the ordering doesn't matter, we might as well make this a set, since the primary use case will be checking for inclusion, and we might one day have many more of these.